imap_processing/tests/spice/test_kernels.py

```diff
@@ -61,9 +93,65 @@ def test_ensure_spice_time_kernels():
 
 def test_ensure_spice_key_error():
     """Test functionality of ensure spice when all branches fail"""
-    wrapped = kernels.ensure_spice(spice.et2utc)
+    wrapped = ensure_spice(spice.et2utc)
     # The ensure_spice decorator should raise a SpiceyError when all attempts to
     # furnish a set of kernels with sufficient coverage for the spiceypy
     # functions that it decorates.
     with pytest.raises(SpiceyError):
         _ = wrapped(577365941.184, "ISOC", 3) == "2018-04-18T23:24:31.998"
+
+
+def test_average_quaternions(et_times, kernels):
+    """Tests average_quaternions function."""
+    # TODO: remove spice.furnsh(kernels) after ensure_spice update.
```

Comment:
Remove this TODO. This is a good way to test this code independent of the `ensure_spice` decorator.